### PR TITLE
unclickable overlay fix on reddit video - #315

### PR DESCRIPTION
**Pull request by canarslan12**
not fixing "in the middle of the video" part

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -95,8 +95,8 @@ chrome.runtime.sendMessage({}, function(response) {
     tc.videoController.prototype.initializeControls = function() {
       var document = this.document;
       var speed = parseFloat(tc.settings.speed).toFixed(2),
-        top = Math.max(this.video.offsetTop, 0) + "px",
-        left = Math.max(this.video.offsetLeft, 0) + "px";
```
Comment (igrigorik) [inline inject.js]: Hmm, why did we need to drop the offset's? 

Is the fix here the special case in the switch below, or the code above?

Comment (canarslan12) [reply inject.js]: They are causing location bug and it's useless if I'm correct

Comment (canarslan12) [reply inject.js]: switch below: fix for unclickable overlay
code above(offset): fix for location of overlay

**Comment by igrigorik** [comment]
Thanks for the nudge.

**Pull request merged**